js/main.js

```diff
@@ -157,64 +157,63 @@ function submit_slide(slide){
   slide.appendChild(element)
 }
 
-function submit_jira(slide){
+function submit_jira(slide) {
```

Comment:
Should this function be in the main.js under the soruce folder? If so, why not just put it there instead of here?